packages/cli/src/utils/misc.js

```diff
@@ -25,7 +25,9 @@ const isWindows = () => {
 
 // Run a bash command with a promise.
 const runCommand = (command, args, options) => {
-  if (isWindows()) {
+  const isRunningUnderWindows = isWindows();
+
+  if (isRunningUnderWindows) {
```

Comment:
nit: Seems like we can combine the other `isRunningUnderWindows` conditional? By either moving `command += '.cmd'` down or moving `options` up?

Reply:
I like it! Thanks for the suggestion ✅